Makefile

```diff
@@ -50,12 +50,15 @@ tdd:
 test_failed:
 	$(DOCKER_PREFIX) py.test --lf $(ARGS)
 
+HIGHEST_MIGRATION = $$(cd migrations; \
+		       ls | sort -rn | sed -n 's/^\([0-9]\+\).*/\1/p' | head -1)
```

Comment:
How about modifying schematic (yeah, I know) to add a parameter that provides the result of running https://github.com/mozilla/schematic/blob/master/schematic#L183?

Or even better, make schematic be a real package that can be installed (and not only a script), so we can `from schematic import find_upgrades`.